wf/ja-jp/Content/Topics/Workflows/connector-builder/connector-builder.htm

```diff
@@ -4,7 +4,7 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <meta name="typeofcontent" content="documentation" />
         <meta name="audience" content="admin workflows" />
-        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><title>Workflowsコネクタービルダー | Okta</title>
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><title>Workflows Connector Builder | Okta</title>
```

Comment:
Regression?

Reply:
In the TOC, this is translated fully. So, if not regression, it is at least inconsistent.
@pengyao-okta ?

Reply:
"Workflows Connector Builder" is DNT term. The change is coming from the global changes.
I think you see other TOC files still using translated terms, that's because we didn't send all the files for this global change.